editor/gui/editor_run_bar.cpp

```diff
@@ -655,13 +683,16 @@ EditorRunBar::EditorRunBar() {
 	write_movie_panel = memnew(PanelContainer);
 	main_hbox->add_child(write_movie_panel);
 
-	write_movie_button = memnew(Button);
+	MenuButton *write_movie_menu_button = memnew(MenuButton);
```

Comment:
Why this isn't `write_movie_button`?

Reply:
Good question... I think it was from me trying to replicate the "Run in Regular/XR Mode" MenuButton code, which works that same way. But since we're always setting `write_movie_button` to the same thing, it's unnecessary here.